app/models/grda_warehouse/hud/user_viewable_entity.rb

```diff
@@ -0,0 +1,8 @@
+module GrdaWarehouse::Hud
+
+  # NOTE, this is a join table where the users are in one database and the entities are in another
+  # there is in fact a belongs_to relationship to the user table, therefore, but it isn't specified here
+  class UserViewableEntity < Base
```

Comment:
No, it's just that all the entities whose visibility we are trying to control are in this module. We just need it to be in the same database.